src/CLI/Command.php

```diff
@@ -97,10 +104,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if (!$count) {
             $output->writeln('No files found to scan');
-            exit(0);
+            return 0;
         }
 
-        $printer = new Text;
+        $printer = new Text();
```

Comment:
Please do not make changes that are not relevant to the purpose of the pull request.

Reply:
Ok, I have extracted the changes to: https://github.com/sebastianbergmann/phploc/pull/176 so if you merge it first it won't show here.

Reply:
The point I wanted to make (and apparently failed to make) is that I do not want those changes.

Reply:
Alright, not sure why not to have brackets, but done.

Reply:
That should make us even, as I do not understand why people want JSON.